docs/developer_documentation/ref-rest2.md

```diff
@@ -81,6 +81,19 @@ GET http://molgenis.mydomain.example/api/v2/<entity_name>?aggs=x==attr0;y==attr1
 GET http://molgenis.mydomain.example/api/v2/<entity_name>?aggs=x==attr0;y==attr1;distinct=attr2&q=attr4==val
 ```
 
+## Including categorical options
+
+For simple lookup lists, it might be a convenience to include the list in the initial API response.
```

Comment:
i would suggest including a description of the the 'default' setting ( not including the param sets it to false) and describing the structure of the response object ( list options with id and label)